modules/kafka/src/test/scala/com/dimafeng/testcontainers/integration/SchemaRegistrySpec.scala

```diff
@@ -27,7 +27,7 @@ class SchemaRegistrySpec extends AnyFlatSpec with ForAllTestContainer with Match
   //a way to communicate containers
   val network: Network = Network.newNetwork()
 
-  val kafkaContainer: KafkaContainer = KafkaContainer.Def(DockerImageName.parse(s"confluentinc/cp-kafka:$kafkaVersion")).createContainer()
+  val kafkaContainer: KafkaContainer = KafkaContainer.Def().createContainer()
```

Comment:
Shouldn't we pin the version? it uses the latest, right?

Reply:
Ah, I had just removed this because I thought it was redundant (and was leading to failures on my machine). It does subtly change the test, but it still passes. Previously it used version `6.1.1` of Kafka (it still uses this version for Schema Registry), now it uses version `7.0.1` (pinned, as defined in the `KafkaContainer` companion object).

Reply:
Ah, I see it now. Thanks for the clarification. We can remove `kafkaVersion` variable then.